app/frontend/src/bulk/components/BulkWellDocuments.vue

```diff
@@ -369,9 +410,11 @@ export default {
       }
     },
     reset () {
+      this.willOverwriteExisting = false
       this.apiError = null
       this.apiValidationErrors = {}
       this.isSaving = false
+      this.unknonwnWellIds = null
```

Comment:
Would it be worth it to fix this typo?

Reply:
:man_facepalming: arg - yes I will fix it! :-D